app/assets/javascripts/discourse/controllers/topic.js.es6

```diff
@@ -116,6 +116,13 @@ export default ObjectController.extend(SelectedPostsCount, BufferedContent, {
     deleteTopic() {
       this.deleteTopic();
     },
+    // Change Timestamp
+    changeTimeStamp(post, dateTime) {
+      if (this.get('selectedPostsCount') === 1) {
```

Comment:
Why is this logic here? It seems like the `post` is always passed to this method. Why would it need to be retrieved again from what is selected?